libs/core/README.md

```diff
@@ -11,29 +11,49 @@ pip install langchain-core
 
 ## What is it?
 
-LangChain Core contains the base abstractions that power the rest of the LangChain ecosystem. 
-These abstractions are designed to be as modular and simple as possible. 
-Examples of these abstractions include those for language models, document loaders, embedding models, vectorstores, retrievers, and more. 
+LangChain Core contains the base abstractions that power the rest of the LangChain ecosystem.\
+
+These abstractions are designed to be as modular and simple as possible. Examples of these abstractions include those for language models, document loaders, embedding models, vectorstores, retrievers, and more.
+
 The benefit of having these abstractions is that any provider can implement the required interface and then easily be used in the rest of the LangChain ecosystem.
```

Comment:
"then easily be used in the rest of the LangChain ecosystem." we should probably paraphrase